lua/autorun/client/cl_ctc_handler.lua

```diff
@@ -205,7 +205,11 @@ function CTC:RunCommand(command)
 	if not self.data.commands[command].console then return end
 
 	for _, line in ipairs(self.data.commands[command].console) do
+		if type(line) == "string" then
```

Comment:
Gmod has the inbuilt function isstring()
```suggestion
		if isstring(line) then
```

Reply:
Or with the other change maybe just check for 
`not istable(line)`
and then transform it into one ^^